openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/AbstractStream.java

```diff
@@ -48,8 +49,15 @@ public void release() {
 
 	public abstract void release(boolean remove);
 
-	public static WebRtcEndpoint createWebRtcEndpoint(MediaPipeline pipeline, Boolean recv) {
+	public static WebRtcEndpoint createWebRtcEndpoint(MediaPipeline pipeline, Boolean recv,
```

Comment:
this constructor injection is not necessary

Reply:
its not a constructor?

Reply:
you are right, parameter here is required